.github/workflows/update-contributors.yml

```diff
@@ -5,9 +5,6 @@ on:
   schedule:
     # run at 4am in the morning on every Monday.
     - cron: '0 4 * * 1'
-  push:
-    branches:
-      - main
 
 jobs:
   update-contributors:
```

Comment:
The expression of "update contributors" is a bit of confusing. You can't update contributors: the persons who have contributed to Asterinas are not determined by this workflow. The workflow only updates the JSON files for the dataset. So I think it should be called "update dataset". The names of this `.yml` file, this workflow, and this job should be fixed.